src/Button/README.md

```diff
@@ -121,22 +121,22 @@ Use inline size buttons for when a button sits with a line of text.
 ### With a Spinner
 ```jsx live
 <>
-  <Button variant="primary" className="mr-2">
+  <Button variant="primary" className="mr-2" aria-label="Spiner">
```

Comment:
[curious] Does adding `screenReaderText` to the `Spinner` component also address the a11y issue?